src/components/templates/Audience/Audience.scss

```diff
@@ -3,8 +3,8 @@
 $light: #ffffff;
 
 // If you change this, make sure to also change it in Audience.tsx's SEAT_SIZE
-$seat-size: 4vh;
-$seat-size-min: 25px;
+$seat-size: max(4vh, 3vw);
+$seat-size-min: 30px;
```

Comment:
Also, why did we change `$seat-min-size` to `30px`? I didn't see any discussions/etc anywhere about that.

Reply:
I don't have an educated answer to this one. I guess my intention was to make seats `larger` so that they would take more space on the small screens.

Reply:
I don't think that's a thing we need to with this. It's only going to apply when `4vh` or `3vw` is smaller than `30px`, which isn't going to be the case on most devices, except for probably mobiles, where I expect we won't want the `30px` as we will need them to be smaller.

---

Not sure why exactly, but this change also seems to make the video window smaller, at least on the current production code:

`--seat-size-min: 25px`:
![image](https://user-images.githubusercontent.com/753891/111402237-f854ae80-871e-11eb-9129-90d511f4d427.png)

`--seat-size-min: 30px`:
![image](https://user-images.githubusercontent.com/753891/111402287-11f5f600-871f-11eb-880f-9b203757ff1b.png)

---

Personally I would just stick to the original plan/investigation, and revert this aspect of this PR. I don't think it adds any benefit, and it seems to detriment things.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1267/commits/3ce92bcfe8cf07d9ad37c72f1fde352e53555bd9